api/clever/__init__.py

```diff
@@ -180,7 +180,7 @@ def remote_patron_lookup(self, token):
             'Authorization': 'Bearer %s' % token
         }
         result = self._get(self.CLEVER_API_BASE_URL + '/me', bearer_headers)
-        data = result.get('data', {})
+        data = result.get('data', {}) or {}
```

Comment:
This is an unrelated change. At one point while testing `patron_email_address` I ended up with a real but misconfigured Clever OAuth provider that made a real request to Clever and got back a dictionary that had `data: None`. If that happens for real, this will at least ensure we get to the point where we can give a good error message.